ui/app/components/generated-item.js

```diff
@@ -48,6 +48,20 @@ export default Component.extend({
   ),
   init() {
     this._super(...arguments);
+    this.model.fieldGroups.forEach((element) => {
+      // overwriting the helpText for Token Polices.
+      // HelpText from the backend says add a comma separated list, which works on the CLI but not here on the UI.
+      // This effects TLS Certificates, Userpass, and Kubernetes. https://github.com/hashicorp/vault/issues/10346
+      if (element.Tokens) {
+        element.Tokens.forEach((attr) => {
```

Comment:
Instead of iterating over all of them can we do `element.Tokens.find(attr => attr.name === 'tokenPolicies')` and then update it if found? That way iteration will stop once the item is found, or have the same behavior as iterating over each if not found

Reply:
Great idea. Amending.